app/appengine_config.py

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/python2.7
```

Comment:
Do you still need this file? It looks working without this file. I believe just fix_path.py is sufficient.